en/docs/learn/running-the-mp-jwt-sample.md

```diff
@@ -105,19 +105,19 @@ First, let's configure the sample. Follow the steps below:
 
     These three endpoints are secured with MP-JWT as follows:
 
-    -   /balance endpoint is allowed to call by users which has one of
+    -   /balance endpoint is allowed to call by users that are assigned one of
         the following roles: admin, ViewBalance, Debtor
```

Comment:
Shall we rephrase these to this format?

The /balance endpoint can be called by users that are assigned to one of the following roles